packages/demo/pages/editable-html.js

```diff
@@ -16,8 +16,7 @@ import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 import InputChooser from '../source/editable-html/input-chooser';
 import { hasText } from '@pie-lib/pie-toolbox/render-ui';
-// import { renderMath } from '@pie-lib/pie-toolbox/math-rendering-accessible';
-import { renderMath as renderMathOld } from '@pie-lib/pie-toolbox/math-rendering';
+import { renderMath } from '@pie-lib/pie-toolbox/math-rendering';
```

Comment:
Shouldn't renderMath be imported from `@pie-lib/pie-toolbox/math-rendering-accessible` instead?

Reply:
This is for testing in the demo page and I made sure that math rendering accessible works with the old math-rendering in https://illuminate.atlassian.net/browse/PD-4527